### PR TITLE
String manipulation - Cynthia Huynh

### PR DESCRIPTION
**Pull request by HuynhCynthia**
Screen Shot 2023-02-03 at 2 03 39 PM

### CODE REVIEW

File: string-manipulation-1/src/is-lower-cased.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  for (let i = 0; i < word.length; i++) {
+    if (word[i] !== word[i].toLowerCase()) {
+      return false;
+    }
+  }
+  return true;
```
Comment (shawnkost) [inline string-manipulation-1/src/is-lower-cased.js:8]: You don't actually need a loop! You can do something like this, and just compare it to the original value
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```

File: string-manipulation-1/src/is-upper-cased.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  for (let i = 0; i < word.length; i++) {
+    if (word[i] !== word[i].toUpperCase()) {
+      return false;
+    }
+  }
+  return true;
```
Comment (shawnkost) [inline string-manipulation-1/src/is-upper-cased.js:8]: Same note as above ^

**Review by shawnkost — APPROVED** [review]
Nice job Cynthia! I left a couple comments on optimizing this! 👍🏻

File: string-manipulation-1/src/is-lower-cased.js (hunk 1 above)
Comment (HuynhCynthia) [reply string-manipulation-1/src/is-lower-cased.js:8]: Ooh right so simple, thank you!

**Pull request merged**